keycloak/keycloak-admin-ui
# Identity Providers(Mappers): Edit mappers

## Motivation


Dependent on #1118 <--- needs merge before this PR is ready for review

## Brief Description

Adds edit functionality to IdP mappers.

## Verification Steps



1. Go to Identity Providers
2. Create/Select an IdP
3. Go to mappers tab
4.  Create/Select an IdP mapper
5. Edit the mapper and verify the data is saved correctly. 

## Checklist:


## Additional Notes
